### DISCUSSION TITLE
readBody for GET requests

### DISCUSSION THREAD

**Discussion by azohc** [nuxt/nuxt#21885] [Questions] [answered]
What is the reason behind `readBody` calls within GET requests resulting in 405 Method Not Allowed HTTP errors?

**Comment by masyoudi** [comment, marked as answer]
because method GET isn't a payload method see https://github.com/unjs/h3/blob/b960a747ba94a67a1f17f06b7c2e430235bc9e3f/src/utils/body.ts#L26
